docs/software/installing/python/cray_python.md

```diff
@@ -0,0 +1,119 @@
+   `PYTHONPATH` environment variable.
+
+You can do this in the following way (once you have set `PYTHONUSERBASE` as described
+above):
+
+    export PATH=$PYTHONUSERBASE/bin:$PATH
+    export PYTHONPATH=$PYTHONUSERBASE/lib/python3.9/site-packages:$PYTHONPATH
+
+Please check that the correct python version (e.g. python3.9) in the above
+command is used.
+
+We would recommend adding all three of these commands to your `$HOME/.bashrc`
+file to ensure they are set by default when you log in to LUMI.
+
+Once, you have done this, you can use `pip` to add packages on top of the HPE
+Cray Python environment. This can be done using:
+
+    module load LUMI/21.12 cray-python
+    pip install --user <package_name>
+
+This uses the `--user` flag to ensure the packages are installed in
+the directory specified by `PYTHONUSERBASE`.
+
+## Setting up virtual environments
+
+Sometimes, you may need several different versions of the same package
+installed, for example, due to dependency issues. Virtual environments allow
+you to manage these conflicting requirements.  We recommend that you use `venv`
+to manage your Python environments. A summary of how to get a virtual
+environment set up is contained in the below, but for further information, see:
+
+   - [Virtual environments with venv](https://docs.python.org/3/tutorial/venv.html)
+
+
+To create a virtual environment, run
+```
+python -m venv --system-site-packages /projappl/project_XXXX/virtual-env
+```
+This will create the `virtual-env` directory if it doesn't exist and install a
+copy of the python interpreter and various other files in there.  Instead of an
+absolute file path (`/projappl/project_XXXX/virtual-env`), a relative path like
+`venv` is also possible. This will create the virtual environment in the
+current folder.
+
+The `--system-site-packages` option allows the use of all system-wide installed
+packages from inside the virtual environment. That on the one hand contradicts
+the intention of virtual environments but allows you to you the specially
+performance tuned CRAY package versions of for example `numpy`, `scipy` and
+`mpi4py`.
+
+Finally, you're ready to `activate` your environment. This is done by running
+```
+source /projappl/project_XXXX/virtual-env/bin/activate
+```
+Once your environment is activated you will be able to install packages as
+usual using `pip install <<package name>>`. These packages will only be
+available within this environment. When running code that requires these
+packages you must activate the environment, by adding the above `source ...
+activate` line of code to any submission scripts.
+
+In case a newer version than the one provided by CRAY is needed you can also
+upgrade it with `pip install -U pandas`. But be aware that the new package
+version will not contain any optimizations for LUMI. So this is only
+recommended if the CRAY provided version is missing features or contains bugs
+fixed in newer versions.
+
```

Comment:
I don't know if this is accurate. Not everything that comes preinstalled with `cray-python` has special cray stuff in it. Numpy in particular, I think it's only compiled against libsci and things that build on it, like pandas, will use that.